### ISSUE TITLE
Fixed / Magically Self-Healed: Doesn't work anymore with latest Opera on Linux

### ISSUE THREAD

**Issue by shundhammer** [gblazex/smoothscroll#202]
I had used this great extension for years, but after upgrading to the latest Opera it stopped working.

I tried resetting all settings, I tried changing all kinds of settings in Opera, I experimented with opera:flags; no change. No matter what values I enter into the extension settings, there is no change; yet the icon on the top status bar says "SmoothScroll has access to this site".

This happens with Opera 69.0.3686.49 based on Chrome 83.0.4103.116 on Xubuntu 18.04.4 LTS.

SmoothScroll has version 1.6.31. I already tried to uninstall and reinstall it; no change.

**Comment by shundhammer** [comment]
I also just tried with a complete stock Opera environment: After closing Opera, I removed `~/.cache/opera`, I renamed `~/.config/opera` to `opera.old`, started Opera again (now with default settings and no extensions at all), added "Smoot Scroll" again; no change. I experimented with different sites and different "Smooth Scroll" settings; no change.

**Comment by shundhammer** [comment]
I am not 100% sure how, but I managed to get it working again.

There was an update to Opera 70.0.3728.71 based on Chrome 84.0.4147.89; with that, it still didn't work.

But while playing around with settings, I found that reverting the scroll direction in the SmoothScroll settings and reverting it back to normal seemed to reset some things, and all of a sudden it worked again.

I still had to tweak some settings to make it work nicely, but that was only optimization:

![opera-flags](https://user-images.githubusercontent.com/11538225/88718239-f7d0be80-d121-11ea-88b0-a75931627587.png)

![smooth-scroll-settings](https://user-images.githubusercontent.com/11538225/88718246-fbfcdc00-d121-11ea-8804-6467b5ffc9f9.png)
